### PR TITLE
Added a flawless comment

### PR DESCRIPTION
**Pull request by jeremycole**
Added a trivial comment with no typos.

### CODE REVIEW

**Review by Caleb-Briggs1 — CHANGES_REQUESTED** [review]
--

File: lib/innodb/page.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@ def lsn_low32
       end
     end
 
+    # This is a really awesome comment and is flwless.
```
Comment (jschneider03) [inline lib/innodb/page.rb:38]: Comment

**Review by jschneider03 — APPROVED** [review]

File: lib/innodb/page.rb (hunk 1 above)
Comment (jeremycole) [inline lib/innodb/page.rb:38]: Yo you left a typo here.

Comment (jeremycole) [inline lib/innodb/page.rb:38]:
```suggestion
    # This is a really awesome comment and is flawless.
```

**Review by jeremycole — COMMENTED** [review]
Looks good but please fix that typo.

**Pull request merged**